src/main/webapp/i18n/de/operatorInfo.json

```
@@ -0,0 +1,10 @@
+{
+    "artemisApp": {
+        "operatorInfo": {
+            "title": "Betreiber",
+            "name": "Name",
+            "admin": "Administrator:in",
+            "adminEmail": "Kontakt Email"
```

```suggestion
            "adminEmail": "E-Mail-Kontakt"
```
or
```suggestion
            "adminEmail": "Kontakt-E-Mail"
```
as compound noun.